src/test/java/edu/tamu/scholars/middleware/discovery/controller/AbstractSolrDocumentControllerTest.java

```diff
@@ -168,6 +168,30 @@ public void testSearchSolrDocumentsCount() throws Exception {
         // @formatter:on
     }
 
+    @Test
+    public void testRecentlyUpdated() throws Exception {
+        // @formatter:off
+        mockMvc.perform(get(getPath() + "/recently-updated").param("limit", "3"))
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
+            .andExpect(jsonPath("$.page.size", equalTo(3)))
+            .andDo(
+                document(
```

Comment:
Please update appropriate asciidocs to reference the generated docs from the test.